include/lsst/afw/image/Image.h

```diff
@@ -505,6 +505,15 @@ void swap(DecoratedImage<PixelT>& a, DecoratedImage<PixelT>& b);
 /// provide the xy0.
 geom::Box2I bboxFromMetadata(daf::base::PropertySet & metadata);
 
+/**
+ * Return true if the pixels for two images or masks overlap in memory
+ *
+ * Images are considered to overlap if they are identical or one is a
+ * subimage of the other.
```

Comment:
What is the desired behavior if two images share pixels, but both have unshared pixels as well? I think this can happen if you have two subimages of the same image.

Reply:
Return true. I'll update the documentation to try to clarify this.

Reply:
Actually I think the simplest thing to do is get rid of the longer explanation and just leave the summary.